gateway/models.py

```diff
@@ -113,10 +111,9 @@ def serialize(self):
             }
         if self.ble_extension:
             response.update(self.ble_extension.serialize())
-        if self.ethermab_extension:
-            response.update(self.ethermab_extension.serialize())
-        if self.fdo_extension:
-            response.update(self.fdo_extension.serialize())
+        for ext in scim_ext_read:
+            ext(self,response)
```

Comment:
Would prefer something like this instead of `ext`. 
```suggestion
        for read_fn in scim_ext_read:
            read_fn(self, response)
```
Same comment for the other places where you have `ext`.